tmp/axum-wasm/src/lib.rs

```diff
@@ -1,6 +1,6 @@
 use tracing::debug;
 
-shuttle_codegen::app! {
+shuttle_next::app! {
     #[shuttle_codegen::endpoint(method = get, route = "/hello")]
```

Comment:
Should we change this to  `shuttle_next::endpoint`?

Reply:
:man_facepalming: Yes